src/shared/components/comment/comment-node.tsx

```diff
@@ -234,6 +234,18 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
             })}
           >
             <div className="d-flex flex-wrap align-items-center text-muted small">
+              <button
+                className="btn btn-sm text-muted mr-2"
+                onClick={linkEvent(this, this.handleCommentCollapse)}
+                aria-label={this.expandText}
+                data-tippy-content={this.expandText}
+              >
+                {this.state.collapsed ? (
```

Comment:
While it was like this before your PR, it could be nice to rewrite lines 243-247 as:
```jsx
<Icon icon={`${this.state.collapsed ? "plus" : "minus"}-square`} classes="icon-inline" />
```

Reply:
Ah, yes. Good suggestion, I'll push the change.